### DISCUSSION TITLE
Got Immediate reject on my application

### DISCUSSION THREAD

**Discussion by akibaso** [community/community#52020] [GitHub Education]
It just reply me:
Unfortunately, we weren't able to approve your educational discount request for [@akibaso](https://github.com/akibaso).

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

I've tried my school id and my admission letter but they didn't work.

**Comment by samus-aran** [comment]
Hi @akibaso 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
